### ISSUE TITLE
Home folder

### ISSUE THREAD

**Issue by rbaldasso** [vivex/cPanel4j#6]
The script is all hardlinked to /home, environments with differents home paths will broke the tomcat deploy.

**Comment by vivekfyi** [comment]
@rbaldasso  , Yup, i got your email last night, so you are looking into it right ?

**Comment by rbaldasso** [comment]
Hi,

Yes, as soon as I can I'll make some improvements on the system. 
- fix blank pages while creating / deleting instances 
- support different home folders
-  display the default tomcat user and password on the plugin
- ability to change the default user and password, also creating new ones

**Comment by vivekfyi** [comment]
cool, let me know if you need any assistance of mine.
